craft.lic

```diff
@@ -0,0 +1,291 @@
+custom_require.call(%w[drinfomon equipmanager common common-crafting common-items common-money common-travel])
+
+class Craft
+  include DRC
+  include DRCC
+  include DRCI
+  include DRCM
+  include DRCT
+
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'craft', options: %w[forging outfitting engineering alchemy], description: 'What craft to train' }
+      ]
+    ]
+    args = parse_args(arg_definitions)
+
+    @settings = get_settings
+    @hometown = @settings.hometown
+    @training_room = (@settings.training_rooms || [@settings.safe_room]).sample
+    @settings.storage_containers.each { |container| fput("open my #{container}") }
+    @equipment_manager = EquipmentManager.new(@settings)
+    @craft_max_mindstate = @settings.craft_max_mindstate
+    @yarn_quantity = @settings.yarn_quantity
+
+    if args.craft == 'engineering'
+      exit if DRSkill.getxp('Engineering') > @craft_max_mindstate
+      train_engineering
+    end
+
+    if args.craft == 'outfitting'
+      exit if DRSkill.getxp('Outfitting') > @craft_max_mindstate
+      train_outfitting
+    end
+
+    if args.craft == 'forging'
+      exit if DRSkill.getxp('Forging') > @craft_max_mindstate
+      train_forging
+    end
+
+    if args.craft == 'alchemy'
+      exit if DRSkill.getxp('Alchemy') > @craft_max_mindstate
+      train_alchemy
+    end
+  end
+
+  def train_outfitting
+    if @settings.train_workorders.include?('Tailoring')
+      return unless money_for_training?(5000, 'Outfitting')
+      wait_for_script_to_complete('workorders', ['Tailoring'])
+      wait_for_script_to_complete('sell-loot')
+      walk_to(@training_room)
```

Comment:
❓ This feels more like the responsibility of `crossing-training`. What do you think?

Reply:
true, also the getting money should be removed from crossing training since workorders handles it directly in a customizable way.

Reply:
it's more here for the future goal of just completely gutting crafting out of crossing-training